osu.Game/Graphics/UserInterface/FPSCounterTooltip.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Cursor;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Platform;
+using osu.Game.Graphics.Containers;
+using osuTK;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class FPSCounterTooltip : CompositeDrawable, ITooltip
```

Comment:
This tooltip shows on single thread execution, providing incorrect details:

<img width="304" alt="CleanShot 2022-07-21 at 02 31 54@2x" src="https://user-images.githubusercontent.com/22781491/180099906-62ccc41a-71aa-44dd-8493-ca1c256374cf.png">

Probably don't want to display a tooltip for single thread, since what's visible in the counter should already be enough info.

Reply:
I've fixed this by removing the maximum frame rate when it's being bypassed for now. Same as what framework display does.